stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -282,9 +282,7 @@ fn test_simple_nakamoto_coordinator_bootup() {
 /// Mine a single Nakamoto tenure with 10 Nakamoto blocks
 #[test]
 fn test_simple_nakamoto_coordinator_1_tenure_10_blocks() {
-    let mut test_signers = TestSigners::default();
-    let mut peer = boot_nakamoto(function_name!(), vec![], test_signers.aggregate_public_key);
-    let private_key = peer.config.private_key.clone();
+    let private_key = StacksPrivateKey::new();
```

Comment:
Why did you change this?

Reply:
`test_simple_nakamoto_coordinator_1_tenure_10_blocks()` and `test_simple_nakamoto_coordinator_10_tenures_10_blocks()` started failing as soon as I added support for pox4 in `pox-locking/src/lib.rs`. Aaron suggested this as a way to get the coordinator tests working again

Reply:
Ah, okay.  I can see why that is -- now that the special-case handlers are in place, stacking would have locked up all of the STX tied to `peer.config.private_key`, which would prevent this test from running (since it sends STX from this key).  Can you at least use a constant private key so the addresses in this test are deterministic?

Reply:
Used Aaron's suggestion of `StacksPrivateKey::from_seed()`